spec/cnb/basic_local_pack_spec.rb

```diff
@@ -1,95 +0,0 @@
-require_relative '../spec_helper'
-
```

Comment:
With this spec removed, the following are unused and can also be removed:
- the cutlass usages in `spec/spec_helper.rb`
- the cutlass dependency can be dropped

(Mentioned in the last review: https://github.com/heroku/heroku-buildpack-ruby/pull/1326#pullrequestreview-1162175333)